ui/store/institutional/institution-background.ts

```diff
@@ -35,6 +35,22 @@ export function showInteractiveReplacementTokenBanner({
     }
```

Comment:
You probably also want to update [builds.yml:125](https://github.com/MetaMask/metamask-extension/blob/cfc653ada651cbcf76e9121a9c95d29cfdb2b202/builds.yml#L125) to `- ./{app,shared,ui}/**/{mmi,institutional}/**` so that these files are not included in normal extension bundle

Reply:
Good catch! Thank you :)